### PR TITLE
Exon ranges

### PR DESCRIPTION
**Pull request by dnil**
This PR adds a functionality and fixes a bug.

**How to test**:
1. look at the SV variants page on stage. make sure you can see exon ranges for a couple of SV variants that should have it (eg an exonic 100kb or so deletion or duplication)
Screenshot 2019-10-11 at 11 48 46

1. look at a SV variant and see that UCSC coordinates are visible. See that you cannot see the button for the whole variant for variants that span across contigs such as translocations (BND).

Screenshot 2019-10-11 at 11 47 51

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by CR
- [x] tests executed by CR

### CODE REVIEW

**Comment by northwestwitch** [comment]
I'm going to review this now!

File: scout/server/blueprints/variants/templates/variants/sv-variant.html [hunk 1]
```diff
@@ -141,6 +141,7 @@
               {% else %}
                 - BAM file(s) missing
               {% endif %}
+              - <a class="btn btn-outline-secondary btn-sm" href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr{{ variant.chromosome }}:{{ variant.position }}&dgv=pack&knownGene=pack&omimGene=pack" target="_blank">UCSC</a>
```
Comment (northwestwitch) [inline scout/server/blueprints/variants/templates/variants/sv-variant.html]: Looking good and the links work:

image

File: scout/server/blueprints/variants/templates/variants/sv-variant.html [hunk 2]
```diff
@@ -223,13 +228,24 @@
...
-      {{ overlapping(overlapping_snvs, variant.rank_score) }}
+    <div class="col-12">
+      <div class="card">
+        <nav>
+          <div class="nav nav-tabs" id="nav-tab" role="tablist">
+            <a class="nav-item nav-link active" id="nav-genes-tab" data-toggle="tab" href="#nav-genes" role="tab" aria-controls="nav-genes" aria-selected="true">Genes</a>
+            <a class="nav-item nav-link" id="nav-transcripts-tab" data-toggle="tab" href="#nav-transcripts" role="tab" aria-controls="nav-transcripts" aria-selected="false">Transcripts</a>
+          </div>
+        </nav>
+        <div class="tab-content" id="nav-tabContent">
```
Comment (northwestwitch) [inline scout/server/blueprints/variants/templates/variants/sv-variant.html]: Maybe you could add mt-3 to this div class, to distantiate the a bit the div from the top?
div class="tab-content mt-3"

File: scout/server/blueprints/variants/templates/variants/sv-variant.html [hunk 3]
```diff
@@ -223,13 +228,24 @@
...
+          <div class="tab-pane fade show active" id="nav-genes" role="tabpanel" aria-labelledby="nav-genes-tab">{{ genes_panel(variant) }}</div>
```
Comment (northwestwitch) [inline scout/server/blueprints/variants/templates/variants/sv-variant.html]: There's a difference in the styles of the 2 tabs: genes_panel and transcript_panel macros:

image

This reflects in 2 different behaviors of the tabs:

image


And 

image


You could maybe adopt the same style for both?

File: scout/server/blueprints/variants/templates/variants/sv-variant.html [hunk 4]
```diff
@@ -410,6 +426,58 @@
   </div>
 {% endmacro %}
 
+{% macro transcripts_panel() %}
```
Comment (northwestwitch) [inline scout/server/blueprints/variants/templates/variants/sv-variant.html]: Super nice to have this!!! 👍

File: scout/server/blueprints/variants/templates/variants/sv-variant.html (hunk 2 above)
Comment (dnil) [reply scout/server/blueprints/variants/templates/variants/sv-variant.html]: :+1:

**Pull request merged**